### PR TITLE
Feature/mouse event module

### PR DESCRIPTION
**Pull request by no-chris**

### CODE REVIEW

File: views/js/core/mouseEvent.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2016 (original work) Open Assessment Technologies SA ;
+ */
+/**
+ * @author Jean-Sébastien Conan <jean-sebastien.conan@vesperiagroup.com>
```
Comment (krampstudio) [inline views/js/core/mouseEvent.js]: add a small description and may be an `@example` tag in the doc.

File: views/js/core/mouseEvent.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        'mouseleave',
+        'mousemove',
+        'mouseout',
+        'mouseover',
+        'mouseup',
+        'show'
+    ];
+
+    /**
+     * Creates an event (requires IE > 9)
```
Comment (krampstudio) [inline views/js/core/mouseEvent.js]: Just for your information, we uses a subset of jsdoc3 (internals conventions) : 
`@return` -> `@returns`, `@function` is not necessary

**Comment by krampstudio** [comment]
Looks good for me. @jsconan will take over the review

File: views/js/core/mouseEvent.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                eventOptions.screenX     || 0,
+                eventOptions.screenY     || 0,
+                eventOptions.clientX     || 0,
+                eventOptions.clientY     || 0,
+                eventOptions.ctrlKey     || false,
+                eventOptions.altKey      || false,
+                eventOptions.shiftKey    || false,
+                eventOptions.metaKey     || false,
+                eventOptions.button      || 0,
+                eventOptions.relatedTarget || null
```
Comment (jsconan) [inline views/js/core/mouseEvent.js]: It should be very useful to have this list of options mentioned in the jsdoc, like:

```
@param {Object} eventOptions - A list of options to precise the event purpose
@param {Boolean} [eventOptions.bubbles] - Allows the event bubbling (default: false)
@param {Boolean} [eventOptions.cancelable] - Event can be prevented (default: false)
@param ...
```

**Pull request merged**